app/views/historic_appointments/_role_appointment.html.erb

```diff
@@ -1,9 +1,9 @@
 <%= content_tag_for :li, role_appointment, class: 'person person-excerpt' do %>
   <div class="inner">
     <div class="image-holder">
-      <%= link_to_if role_appointment.has_historical_account?, role_appointment.person.image, historic_appointment_path(role.historic_param, role_appointment.person), class: 'img' %>
+      <%= link_to_if role_appointment.has_historical_account?, role_appointment.person.image, historic_appointment_path(role.historic_param, role_appointment.person), class: "img", aria: {hidden: true}, tabindex: "-1" %>
     </div>
-    <h3 class="name"><%= link_to_if role_appointment.has_historical_account?, role_appointment.person.name, historic_appointment_path(role.historic_param, role_appointment.person) %></h3>
+    <h3 class="name"><%= link_to_if role_appointment.has_historical_account?, role_appointment.person.name, historic_appointment_path(role.historic_param, role_appointment.person), class: "govuk-link" %></h3>
```

Comment:
Can we adjust the line height to avoid the focus lower border being partially obscured?

A `line-height` of 1.2 gives this result in Safari:
<img width="239" alt="Screenshot 2019-11-05 at 10 45 27" src="https://user-images.githubusercontent.com/1732331/68201297-67fa4b80-ffb9-11e9-9c9a-a1d8ef7628dc.png">
 
This'll need checking in other browsers. I found a [similar thing happened for related links](https://github.com/alphagov/govuk_publishing_components/pull/1045) and the behaviour varied from browser to browser. Unfortunately those numbers don't _quite_ work here.

Reply:
Changed to 1.16 where it works in Chrome, Firefox, IE11 and Safari!

Reply:
Great work!